[2023-02-14 13:03:14,801] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 12_manually_defined_dag.task_2 manual__2023-02-14T19:03:11.595357+00:00 [queued]>
[2023-02-14 13:03:14,807] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 12_manually_defined_dag.task_2 manual__2023-02-14T19:03:11.595357+00:00 [queued]>
[2023-02-14 13:03:14,807] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-14 13:03:14,808] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-02-14 13:03:14,808] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-14 13:03:14,817] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): task_2> on 2023-02-14 19:03:11.595357+00:00
[2023-02-14 13:03:14,820] {standard_task_runner.py:52} INFO - Started process 397961 to run task
[2023-02-14 13:03:14,822] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', '12_manually_defined_dag', 'task_2', 'manual__2023-02-14T19:03:11.595357+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/Apache_Airflow/12_manually_defined_dag.py', '--cfg-path', '/tmp/tmpi7sm8cnl', '--error-file', '/tmp/tmps0yo4yzn']
[2023-02-14 13:03:14,823] {standard_task_runner.py:80} INFO - Job 203: Subtask task_2
[2023-02-14 13:03:14,876] {task_command.py:369} INFO - Running <TaskInstance: 12_manually_defined_dag.task_2 manual__2023-02-14T19:03:11.595357+00:00 [running]> on host Tinmar.
[2023-02-14 13:03:14,931] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=12_manually_defined_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2023-02-14T19:03:11.595357+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-14T19:03:11.595357+00:00
[2023-02-14 13:03:14,932] {logging_mixin.py:115} INFO - {}
[2023-02-14 13:03:14,933] {python.py:173} INFO - Done. Returned value was: None
[2023-02-14 13:03:14,943] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=12_manually_defined_dag, task_id=task_2, execution_date=20230214T190311, start_date=20230214T190314, end_date=20230214T190314
[2023-02-14 13:03:14,954] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-14 13:03:14,979] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
